src/styles/blocks/header.scss

```diff
@@ -0,0 +1,15 @@
+// --text-color: #00acdc;
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  background: #fff;
```

Comment:
Consider using variables for repeating colors

Reply:
Done!